consensus/consensus.go

```diff
@@ -31,8 +38,29 @@ var (
 
 // Consensus is the main struct with all states and data related to consensus process.
 type Consensus struct {
-	phase PbftPhase
-	mode  PbftMode
+	ConsensusVersion string
+	phase            PbftPhase
```

Comment:
Please add comment for phase, mode, and seqNum

Reply:
added